sql/updates/world/2016_99_99_99_world.sql

```diff
@@ -0,0 +1,8 @@
+DELETE FROM gossip_menu_option WHERE menu_id=11017;
+INSERT INTO `gossip_menu_option` (`menu_id`, `id`, `option_text`, `OptionBroadcastTextID`, `option_id`, `npc_option_npcflag`) VALUES 
+(11017, 0, 'Teleport to Light\'s Hammer.', 37671, 1, 1),
+(11017, 1, 'Teleport to the Oratory of the Damned.', 37722, 1, 1),
+(11017, 2, 'Teleport to the Rampart of Skulls.', 37723, 1, 1),
+(11017, 3, 'Teleport to the Deathbringer\'s Rise.', 37724, 1, 1),
+(11017, 4, 'Teleport to the Upper Spire.', 37725, 1, 1),
+(11017, 5, 'Teleport to Sindragosa\'s Lair.', 37728, 1, 1);
```

Comment:
I know that's why I ask the help. As you can see there are different going and what they are.